test/command/4677.md

```diff
@@ -2,8 +2,5 @@
 % pandoc --to "markdown-bracketed_spans-fenced_divs-link_attributes-simple_tables-multiline_tables-grid_tables-pipe_tables-fenced_code_attributes-markdown_in_html_blocks-table_captions-smart"
 ![Caption](img.png){#img:1}
 ^D
-<figure>
 <img src="img.png" id="img:1" alt="Caption" />
-<figcaption aria-hidden="true">Caption</figcaption>
-</figure>
```

Comment:
Why isn't this an "implicit_figure"?

Reply:
This can't be an implicit figure as we'd loose due to `link_attributes` being disabled. But it should still be a figure. Fixed now.